docs/pages/usage/peering.md

```diff
@@ -1,31 +1,87 @@
 ---
-title: Peering
-weight: 4
+title: Peer with a cluster
+weight: 1
 ---
 
 ## Overview
 
-The last step to perform to join two clusters is peering. It allows establishing a connection between the two 
-clusters and to exchange resources.
+In Liqo, peering establishes an administrative connection between two clusters and enables the resource sharing across them.
+It is worth noticing that peering is unidirectional. 
+This implies that resources can be shared only from a cluster to another and not the vice-versa. Obviously, it can be optionally be enabled bi-directionally, enabling a two-way resource sharing.
 
-## Enable peering
+### Peer with a new cluster
 
-The LAN and DNS discovery have the autojoin feature set by default, i.e., once the clusters are discovered and
-authenticated, the peering happens automatically. If the foreign cluster has been added through a manual configuration,
-you can enable the peering by setting its join flag as follows:
+To peer with a cluster, you should provide some information to Liqo in order to reach that cluster.
+*liqoctl* can help you by generating the command to peer with a specific cluster.
+
+The installation of *liqoctl* is quick and you can find the complete instructions in a [dedicated section](/installation#liqoctl)
+
+Let's suppose that you want to peer from cluster A to cluster B.
+This means that we would like to offload pods in A to the cluster B.
+
+#### Generate the peer command
+
+First, pointing cluster B:
```

Comment:
I did not understand the meaning of "pointing cluster B". Are you still in cluster A?